src/main/kotlin/no/fdk/concept_catalog/service/ImportService.kt

```diff
@@ -0,0 +1,188 @@
+package no.fdk.concept_catalog.service
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import no.fdk.concept_catalog.model.*
+import no.fdk.concept_catalog.rdf.extract
+import no.fdk.concept_catalog.repository.ConceptRepository
+import no.fdk.concept_catalog.repository.ImportResultRepository
+import org.apache.jena.rdf.model.Model
+import org.apache.jena.rdf.model.ModelFactory
+import org.apache.jena.rdf.model.Resource
+import org.apache.jena.riot.Lang
+import org.apache.jena.shared.JenaException
+import org.apache.jena.vocabulary.RDF
+import org.apache.jena.vocabulary.SKOS
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.http.HttpStatus
+import org.springframework.security.oauth2.jwt.Jwt
+import org.springframework.stereotype.Service
+import org.springframework.web.server.ResponseStatusException
+import java.io.StringReader
+import java.time.LocalDateTime
+import java.util.*
+
+@Service
+class ImportService(
+    private val historyService: HistoryService,
+    private val conceptRepository: ConceptRepository,
+    private val importResultRepository: ImportResultRepository,
+    private val objectMapper: ObjectMapper
+) {
+    fun importRdf(catalogId: String, concepts: String, lang: Lang, user: User, jwt: Jwt): ImportResult {
+        val model: Model
+
+        try {
+            model = ModelFactory.createDefaultModel().apply {
+                read(StringReader(concepts), "https://example.com", Lang.TURTLE.name)
```

Comment:
Er kanskje bedre å ikke legge til example.com og i stedet bare feile?
```suggestion
                read(StringReader(concepts), Lang.TURTLE.name)
```